### DISCUSSION TITLE
Does node sync affect plotting times?

### DISCUSSION THREAD

**Discussion by proud2blds** [Chia-Network/chia-blockchain#5618] [Q&A]
Hi all, does anyone know of node syncing affects plotting times. I have a Dell T5810 that hasn't finished syncing yet with xeon e52699v3 64 ddr4 4tb inland platinum nvme. Plotting in parallel 9 plots, 5 threads, 60 min stagger, max 4 in phase 1 and it takes 13 to 15 hours per plot. I have another i710700k similar specs fully synced and only takes about 8 hours.

**Comment by dvd101x** [comment]
That's odd, It shouldn't affect plotting as plotting does not require an internet connection.
